### ISSUE TITLE
erroneous toolbar separator positions using Tabs

### ISSUE THREAD

**Issue by kriete** [bokeh/bokeh#4787]
Hey,

I was plotting some bokeh graphs using Panels and Tabs and came along some strange behaviour at the toolbar separator positions (vertical grey lines).

Using no Tabs
![no_tabs](https://cloud.githubusercontent.com/assets/15364653/16730485/829963d8-4773-11e6-99a5-51a500ffe09e.png)

Using Tabs
![tabs](https://cloud.githubusercontent.com/assets/15364653/16730484/8298d602-4773-11e6-8883-bddf21f97054.png)

You see on the right side the the vertical line separators which should be besides the Tools, but are below?

I am providing a part of my code base as a more or less minimal example code:

```
from bokeh.models.widgets import Panel, Tabs
from bokeh.plotting import figure, output_file, show
x = [1, 2, 3, 4, 5]
y = [6, 7, 2, 4, 5]
p = figure(plot_width=1200, plot_height=300, tools=["pan, xwheel_zoom, hover, reset"],
               y_range=(1, 8), y_axis_label='asdasd', toolbar_location='above', logo=None,
               active_scroll='xwheel_zoom')
p.line(x, y, line_width=2)
p1 = figure(plot_width=1200, plot_height=300, tools=["pan, xwheel_zoom, hover, reset"],
               y_range=(1, 8), y_axis_label='asdasd', toolbar_location='above', logo=None,
               active_scroll='xwheel_zoom')
p1.line(x, y, line_width=2)
t = Tabs(tabs=[Panel(child=p, title='asd'), Panel(child=p1, title='asdasd')])
output_file('test.html')
show(t)
```

bokeh info: Python version      :  2.7.6 (default, Jun 22 2015, 17:58:13) 
IPython version     :  4.1.2
Bokeh version       :  0.12.0
No traceback or browser console output logged.

Would be great if someone knows how to fix this behaviour,
Andreas

**Comment by birdsarah** [comment]
Yep, confirmed this is a bug. It's a subtle css issue that I haven't been able to find a quick fix for. 

The only work around I've found so far is to manually add the following css:

```css
.bk-root .bk-button-bar-list > li:last-child::after {
    display: none
}
```

**Comment by kriete** [comment]
Hey Sarah,

okay thank you for the info and also for the fast response!
Just as perhaps another note for that: I haven't had this behaviour while using bokeh v0.10. Maybe that also helps to track the bug down.

Regards,
Andreas

**Comment by mattpap** [comment]
This was fixed in PR #7008.
